app/controllers/application_controller.rb

```diff
@@ -1,11 +1,44 @@
 class ApplicationController < ActionController::API
 
```

Comment:
A file overview here would be nice, is this a controller for the entire application?